packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_AlgebraicOverlappingOperator_def.hpp

```diff
@@ -94,11 +94,11 @@ namespace FROSch {
             << " |"
             << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
             << "| " << left << setw(41) << "Solver type" << right
-            << " | " << setw(41) << this->ParameterList_->sublist("Solver").get("SolverType","Amesos")
+            << " | " << setw(41) << this->ParameterList_->sublist("Solver").get("SolverType","Amesos2")
```

Comment:
@mayrmt Thank you for this suggestion. I do agree that this should be done at some point. However, since this should be done consistently throughout FROSch, I would like to postpone this for a bit.